tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compileOnSave": true,
+  "compilerOptions": {
+    "target": "es2018",
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "outDir": "./dist",
+    "sourceMap": true,
+    "suppressImplicitAnyIndexErrors": true,
```

Comment:
Personally I prefer strict typings. 

I would be tempted to go in the other direction and add:
```js
"alwaysStrict": true,
"noImplicitAny": true, // you explicitly add any to type signatures if needed
"strictNullChecks: true,
```
https://www.typescriptlang.org/tsconfig#Strict_Type_Checking_Options_6173

Reply:
definitely - I've never been able to enforce these on the projects we run due to having mixed js and typescript and gradual adoption but that has been the end goal. 

I think there is single strict option that covers all the separate flags.